ArduPlane/GCS_Mavlink.cpp

```diff
@@ -235,13 +235,15 @@ void GCS_MAVLINK_Plane::send_position_target_global_int()
 
 float GCS_MAVLINK_Plane::vfr_hud_airspeed() const
```

Comment:
It would be nice to move this function up as well. See the comment [here](https://github.com/ArduPilot/ardupilot/pull/18813#discussion_r720757567) and code below about the difference between copter & plane. 

It would be nice if the plane & copter version of this could be reconciled and moved up.

**Copter**
https://github.com/ArduPilot/ardupilot/blob/eb9b29f84b940553a2482abca8c1ae3bf2d33542/ArduCopter/GCS_Mavlink.cpp#L196-L204

**Plane**
https://github.com/ArduPilot/ardupilot/blob/eb9b29f84b940553a2482abca8c1ae3bf2d33542/ArduPlane/GCS_Mavlink.cpp#L236-L254